accepted/0000-workspaces-command.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-04
+- RFC PR:
+
+# Summary
+
+This document specify a new command, that can be used to execute subcommands inside a workspace projects.
+
+# Motivation
+
+With the addition of workspaces, it will become handy to be able to execute commands inside other projects than the one in the current directory.
```

Comment:
There are some specific scenarios that we need to address in this RFC.

For example, from my RFC #60

The structure of the source code is following

```
| jest/
| ---- package.json
| ---- packages/
| -------- babel-jest/
| ------------ packjage.json
| -------- babel-preset-jest/
| ------------ packjage.json
...
```

We want to address how we run commands (install/add/upgrade/remove) when CWD is workspace root.
Then what happens if CWD is jest/packages/babel-preset-jest?

How do I update a dependency in a workspace and in a root?